src/common/plugin/pluginState.ts

```diff
@@ -138,7 +139,7 @@ export class PluginState extends ServerCache {
     // this.printServerCache()
     if (this.serverCacheDirty) {
       try {
-        await this.disklet.setText(
+        await this.memlet.setJson(
```

Comment:
`setJson` expects an object for its data argument. `JSON.stringify` is already invoked internally.